daml-lf/spec/transaction.rst

```diff
@@ -300,6 +302,17 @@ As of version 1, these fields are included:
 Every element of ``stakeholders`` is a party identifier.
 ``signatories`` must be a non-empty subset of ``stakeholders``.
 
+TODO how to indicate that the two sentences above are not a property
+of the wire format, but rather a description of the ledger semantics
+of those fields? Should we maybe put these docs somewhere else?
```

Comment:
I think they're nice, so in order of preference:

1. direct link to elsewhere describing the semantics (no further details stated here);
2. if linking impossible/unreliable, set here in a `.. note:: *This note is non-normative.*` or some such.

Reply:
i went for number 2.